pandas/tests/frame/conftest.py

```diff
@@ -40,6 +40,18 @@ def int_frame():
     return DataFrame({c: s for c, s in compat.iteritems(df)}, dtype=np.int64)
 
 
+@pytest.fixture
```

Comment:
so don't add this here, rather make it a fixture in test_apply.py; this conftest is for pretty common fixtures, this is not one of them (of course if could decide later to change this)